src/services/swap_service.ts

```diff
@@ -208,6 +208,7 @@ export class SwapService {
             sources: serviceUtils.convertSourceBreakdownToArray(sourceBreakdown),
             orders: serviceUtils.cleanSignedOrderFields(orders),
             allowanceTarget,
+            quoteReport,
```

Comment:
⚠️ Need to make sure this never gets returned to the end user ⚠️

Reply:
You did this, over in the handler, with the `cleanedQuote`, so this comment can be resolved, right?

Reply:
You did this, over in the handler, with the `cleanedQuote`, so this comment can be resolved, right?

Reply:
Correct 👍